src/command/init/transitions.rs

```diff
@@ -355,12 +391,35 @@ impl CreationConfirmed {
             Err(RoverClientError::GraphCreationError { msg })
                 if msg.contains("Service already exists") =>
             {
-                return Err(RoverError::new(anyhow!(
-                    "Graph ID is already in use. Run {} again with a different graph ID.",
-                    Style::Command.paint("`rover init`")
-                )));
+                println!();
+                println!();
+                println!(
+                    "{} Graph ID is already in use. Please try again with a different graph ID.",
+                    Style::Failure.paint("Error:")
+                );
```

Comment:
nit: instead of `println` and having to add our own `Error:`, we can use the `errln!` macro that will do this for us & will help keep our error messaging style consistent across the experience
```suggestion
                errln!(
                    "Graph ID is already in use. Please try again with a different graph ID."
                );
```